integration-tests/src/test/java/uk/gov/di/authentication/api/TokenIntegrationTest.java

```diff
@@ -61,7 +78,8 @@ public void shouldCallTokenResourceAndReturn200() throws JOSEException {
         RedisHelper.addAuthCodeAndCreateClientSession(
                 code, "a-client-session-id", TEST_EMAIL, generateAuthRequest().toParameters());
         Map<String, List<String>> customParams = new HashMap<>();
-        customParams.put("grant_type", Collections.singletonList("authorization_code"));
+        customParams.put(
```

Comment:
where is 'customParams' used? doesn't seem to get passed to the request or checked in the response.

Reply:
On line 87